pkgs/top-level/python-packages.nix

```diff
@@ -7229,7 +7231,7 @@ in {
 
   sexpdata = callPackage ../development/python-modules/sexpdata { };
 
-  sfepy = callPackage ../development/python-modules/sfepy { };
+  sfepy = disabledIf (!pythonAtLeast "3.8") (callPackage ../development/python-modules/sfepy { });
```

Comment:
```suggestion
  sfepy = callPackage ../development/python-modules/sfepy { };
```

Reply:
Done. Makes more sense to keep the logic in `default.nix`.